app/views/admin/task.scala.html

```diff
@@ -6,17 +6,27 @@
     @navbar(admin)
     <div class="container">
         <div class="row">
-            <div class="col-lg-12">
-                <div id="map"></div>
+            <div class="col-md-8 task-map">
+                <div id="map" style="max-width: 768px;"></div> <!--style="position: relative; width: 100%;"-->
+            </div>
+            <div class="col-md-4">
                 <div id="svholder"></div>
-                <div id="eventsholder"></div>
                 <div id="timeline">
                     <div id="timeline-active"></div>
                     <div id="timeline-handle"></div>
                 </div>
+                <div id="eventsholder"></div>
             </div>
         </div>
     </div>
+    <script type="text/javascript" src='@routes.Assets.at("javascripts/lib/bootstrap-3.3.5/js/bootstrap-slider.min.js")'></script>
+    <link href='@routes.Assets.at("javascripts/lib/bootstrap-3.3.5/css/bootstrap-slider.min.css")' rel="stylesheet"/>
+    <script>
```

Comment:
I was fiddling around with integrating the bootstrap slider to replace the css slider we currently have on the playback page. I think I shifted around some javascripts in the process and forgot to revert those changes.